scripts/pop_demix.py

```diff
@@ -1,22 +1,20 @@
 from neurophotonics.demix import *
+from scripts.helper import keys_used
 
-try:
-    IlluminationCycle.populate(reserve_jobs=True, display_progress=False, processes=1024)
-except ValueError:
-    print("IlluminationCycle already populated")
-
-try:
-    Demix.populate(display_progress=False, reserve_jobs=True, processes=1024)
-except ValueError:
-    print("Demix already populated")
-
-try:
-    Cosine.populate(reserve_jobs=True, display_progress=False, processes=1024)
-except ValueError:
-    print("Cosine already populated")
-
-
-try:
-    SpikeSNR.populate(reserve_jobs=True, display_progress=False, processes=1024)
-except ValueError:
-    print("SpikeSNR already populated")
+print(
+    "keys processed:",
+    keys_used(
+        IlluminationCycle.populate(
+            reserve_jobs=True, processes=1024, display_progress=False
+        )
+    )
+    + keys_used(
+        Demix.populate(display_progress=False, processes=1024, reserve_jobs=True)
+    )
+    + keys_used(
+        Cosine.populate(reserve_jobs=True, display_progress=False, processes=1024)
+    )
+    + keys_used(
+        SpikeSNR.populate(reserve_jobs=True, display_progress=False, processes=1024)
+    ),
```

Comment:
These are very fast computations, so I do not see a benefit in multiprocessing.

Reply:
There shouldn't be a heavy cost associated with either option, but I went ahead and disabled multi-processing for these tables.